src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -19,6 +21,8 @@ namespace ov {
 namespace intel_cpu {
 namespace node {
 
+static constexpr IShapeInfer::port_mask_t port_mask = PortMask(/*BEGIN_ID*/1, /*END_ID*/2, /*STRIDE_ID*/3, /*AXES_ID*/4);
```

Comment:
I did not notice this before. Could you please move this constant and all the shape inference related classes into an anonymous namespace, so that they will not be accessible outside the translation unit. In that case the `static` qualifier may be omitted.
```cpp
namespace {
constexpr IShapeInfer::port_mask_t port_mask = PortMask(/*BEGIN_ID*/1, /*END_ID*/2, /*STRIDE_ID*/3, /*AXES_ID*/4);
class StridedSliceShapeInfer : public ShapeInferEmptyPads {
...
};

class StridedSliceShapeInferFactory : public ShapeInferFactory {
...
};
} // namespace
```